### DISCUSSION TITLE
915mhz antenna options? LORA/RNode

### DISCUSSION THREAD

**Discussion by pizzadoglover911** [markqvist/Reticulum#258] [General]
Good day,

My initial range testing revealed relitivley low performance, from what I was expecting. 1,414.62 ft (431.17 m) from radio to radio through medium denisty forest with low noise floor. I think I am being limited by the antennas(and further RNode config). 

I am looking for replacement antennas, does anyone have any experience getting the best range out of these radios? Any experience with antennas sold for[ helium mining](https://www.amazon.com/Hexa-Boost-Antenna-Outdoor-Omni-Direction/dp/B0953Z1QF3/ref=sxin_18_ac_d_bv?ac_md=3-2-QmV0d2VlbiAkNTAgYW5kICQ3NQ%3D%3D-ac_d_bv_bv_bv&content-id=amzn1.sym.8f2bf95d-b9c2-4e6d-96a9-5fdf77a1951d%3Aamzn1.sym.8f2bf95d-b9c2-4e6d-96a9-5fdf77a1951d&cv_ct_cx=915mhz+antenna&keywords=915mhz+antenna&pd_rd_i=B0953Z1QF3&pd_rd_r=1d30ed98-902b-4c21-b49e-7aee2b8bcd2a&pd_rd_w=I7ks9&pd_rd_wg=2hqLo&pf_rd_p=8f2bf95d-b9c2-4e6d-96a9-5fdf77a1951d&pf_rd_r=FAMVB9FKQYRTYTMVH6CN&qid=1678588750&sbo=RZvfv%2F%2FHxDF%2BO5021pAnSA%3D%3D&sr=1-3-270ce31b-afa8-499f-878b-3bb461a9a5a6)? There are others listed about but Im not sure performance is accepetable. 

What antennas are being used here? Thanks!

**Comment by markqvist** [comment]
What boards and antennas were you using? And which LoRa parameters? With that info, we can calculate an estimated link budget to put things into perspective, and from that see what the best options are in terms of improvement :)

**Reply by pizzadoglover911** [reply]
Heltec v2 boards with the provided antennas. I believe I am being limited by these antennas. I cannot prove it without spectrum analyzer, so my hope was to find commercially available tested options.

I know more about lora parameters now, I can't recall what I had set at that time. Now I am set to 250khz band, 11 SF, 6 coding. 800 kps(I can't recall the exact number).

I just tried for a long haul, 5miles with line of sight. No luck. Unfortunately my local topography means line of sight is often wide ranges and long hikes. At least without an assistant and while the waterways are thawing.

**Reply by markqvist** [reply]
Five miles with line of sight should be no problem at all. There is definitely something strange going on then. Your suspicion about the antennas might be correct.

**Comment by Dant3s** [comment]
Hi :wave:  I have the same doubt:

In my case the idea is in the outside of a City have signal with someone in the city. Both are using Heltec v2. So far with a 2dBi antenna but those for 860/2100mhz of 10cm. 

In my specs im using 915/125/tx@10/Spreading@10/Coding@8  in both stations. With those settings what is the expected theoretical outcome in distance?

**Comment by faragher** [comment]
Two major questions you need to answer for yourself: what's your use case, and what's your local legal situation. Assuming you're in the US (due to the 915 MHz) you can push your tx power pretty high over spread spectrum, but I am not a lawyer and cannot provide legal advice. [This](https://www.digikey.com/en/articles/unlicensed-915-mhz-band-fits-many-applications-and-allows-higher-transmit-power) indicates 36 dBm maximum, but I'd stay under 25-30 just because our equipment isn't precise enough to redline without accidentally breaking Federal law.

Power from the transmitter and gain from the antenna make up this broadcast power. There is absolutely nothing magical about an antenna. They provide gain by limiting broadcast area. If you broadcast to half the area, you double your effective power. This assumes a good antenna, but a half wavelength antenna is only about 16cm/6in, so we're likely all running some pretty stock 1/4 wavelength mass produced antennas.

So, think about what you want. If you're looking for talking to someone moving on the ground from a stationary point, you don't need to be broadcasting into the sky or into the ground. This is why most simple antennas are "3dB Omnidirectional," or they broadcast in 360 degrees, but not up and down (assuming a vertical antenna). 

You can see lots of further information on folded dipoles and the like [here](https://www.antenna-theory.com/antennas/main.php). I do not recommend rolling your own, but it helps to shop.

If you want maximum point to point communication, parabolic antennas are the way to go, or phased arrays. Remember that a receiving antenna increases the gain on top of the transmitting antenna, particularly parabolic antennas. However, cheap antennas like Yagis will be rather large at 915 MHz (around 6 feet long or so). 

Also, one of the most important things is line of sight. If you can elevate the antenna above terrain, you need less power to get a good signal.

If you're looking at two mobile stations at ground level through trees, I'd be pleased with 1,500 feet. If either station isn't mobile, elevate it and get a directional antenna. If neither station is mobile, use directional antennas on both stations with as close to clear LOS as you can get. 

I'm not really skilled in RF theory or antenna design, but hopefully this should be helpful. At the very least you can think about what kind of antenna you need and if simply elevating a half or full wave monopole is all you need. And remember, 3dB is double the signal strength, and doubling the range will, in theory, require eight times the power, or 9dB. Since any good antenna will tell you the gain and radiation pattern, that should let you estimate commercial designs.

**Reply by pizzadoglover911** [reply]
Thank you for the reply and knowledge. Your explanation is clear.

I suppose my next issue is determining reputable antenna suppliers/mfgs. While I understand that high gain claims on Omni antennas can be frivolous, it is otherwise hard to discern quality/functional antennas from otherwise plastic covered junk.

**Comment by pizzadoglover911** [comment]
For others who read this and may find it helpful, I found @ricinnewmexico posted about antenna results in relation to meshtastic. Lora just the same, so I figure I will post that link here.

https://github.com/RicInNewMexico/Meshtastic-Antenna-Reports

Maybe this list is something worth contributing to or make a greater note of when discussing hardware. While I understand the entry level equipment has come down in price, it doesn't seem practical to assume all users would have frequency analyzers and the knowledge to test and modify antennas. 

There are only so many frequencies across the lora spectrum, seems there could be more concrete information on which antennas are effective for our use cases.

**Comment by faragher** [comment]
I was unexpectedly called out yesterday and threw an RNode on my laptop to test a couple points on the way. Sadly the only points I checked were failures, one at 1.5km and another at 2.15km. Both had no intervening terrain, but some woodland and residential in between. While these are non-trivial distances, and one of the RNodes was nestled in an electronics bay and the other in a car (2.15 km also tested outside the vehicle) SF8, CR5. 

I intend to test new antennas and have a more rapid GPS test in the near future, as well as possibly cannibalizing my Project 25 system to get another dipole freed up (or just order a new one direct from RTL-SDR, as Amazon is dry) but I currently thing the little antennas that come with the Lilygos simply aren't ideal for long range communication.

**Comment by faragher** [comment]
Trying to get a SWR for 915 MHz is a pain. Sitting in a box full of EMI doesn't help, and apparently the RTL-SDR extension cables are poorly shielded enough to get huge interference effects. I can't get good readings, even with some minor attempts at shielding. 

However, I found something useful. I *think* it can get a SWR of 2 or better, and they're dirt cheap. I used it for a base station for field tests, and I got what I'd consider reasonable results. https://lowpowerlab.com/shop/product/193

They also make a 868MHz model as well, and they can both be trimmed for fine tuning. I'll likely use them as my default from here on out. If you're going to try it, bear in mind they are directional, and should be mounted vertically (I have the ground arm downward, probably doesn't matter) and it should get you about 2dB of gain. If used on the transmitter and receiver, that's a free 4dB. In theory. A lot of field tests right now, and I'm not fond of how loose my methodology is, so it's basically getting bad data to develop a way to find good data.
